gravitee-apim-console-webui/src/management/api/policy-studio/debug/models/DebugResponse.spec.ts

```diff
@@ -247,6 +254,7 @@ describe('convertDebugEventToDebugResponse', () => {
     id: 'request-input',
     duration: 0,
     status: undefined,
+    stage: undefined,
```

Comment:
Really necessary ? it wouldn't be `UNDEFINED` ?

Reply:
good question ^^